src/Sylius/Bundle/ApiBundle/SectionResolver/AdminApiSection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\SectionResolver;
+
+use Sylius\Bundle\CoreBundle\SectionResolver\SectionInterface;
+
+class AdminApiSection implements SectionInterface
```

Comment:
I'm not sure if it is not misleading in comparison to old AdminAPI

Reply:
I thought about that, but it is still the most significant name and it is in different bundle, so I think it is the best name after all.